test/support/utils.rb

```diff
@@ -38,8 +38,6 @@ def debuggee_backlog test_info
 
     # This method will execute both local and remote mode by default.
     def debug_code(program, boot_options: '-r debug/start', remote: true, &block)
-      check_line_num!(program)
```

Comment:
Why did you move this one?

Reply:
I think `test/assertion_helper_test.rb` and `test/test_utils_test.rb` are not the main use case for line numbers?
Anyway, I'm not against checking line numbers on them either. But currently I can't adjust my test script easily before they pass because of line numbers. And that's quite frustrating 🙁

Reply:
I've removed this change as well. But can you find a way to let line checking happen after tests? Otherwise I need to make this change locally every time to I can finish my test script first.

Reply:
Thanks!
`test builder` is fastest way to solve the problem, but you don't wanna use it, right?

Reply:
IMO, test builder should not be a replacement of writing tests in the old fashion way 🙂 
Writing test script in a separate file, run cli commands, and go back to test file to finish the test is just too much switching. 

And even if I do that, I still can't easily modify the test script because it has line numbers. (I know in this case I should update the original file again, but that's confusing).